src/app/speechbar.service.ts

```diff
@@ -124,6 +124,14 @@ export class SpeechbarService {
     }
   }
 
+  sayButton(button: Button) {
+    // we do want to queue these up, so no speaking check
+    // we also want the speak button to stay enabled for queing up the actual message, so no listener updates
+    const msg = new SpeechSynthesisUtterance();
```

Comment:
Might be an idea to pull this out into a factory method as I think it's this that will be configured with voices/pitch etc